disperser/batcher/grpc/dispatcher.go

```diff
@@ -153,28 +156,35 @@ func getBlobMessage(blob *core.BlobMessage) (*node.Blob, error) {
 		lengthProofData.YA1 = blob.BlobHeader.LengthProof.Y.A1.Marshal()
 	}
 
-	quorumHeaders := make([]*node.BlobQuorumInfo, len(blob.BlobHeader.QuorumInfos))
+	quorumHeaders := make([]*node.BlobQuorumInfo, 0)
```

Comment:
Would it better just to revert the changes here?

Reply:
Sure!